mmdet/datasets/custom.py

```diff
@@ -63,6 +63,7 @@ def __init__(self,
                  data_root=None,
                  img_prefix='',
                  seg_prefix=None,
+                 seg_map_suffix='.png',
```

Comment:
seg_prefix and seg_map_suffix seems to be inconsistent. seg_suffix might be better.